src/applications/disability-benefits/all-claims/tests/property-names.unit.spec.js

```diff
@@ -55,7 +60,7 @@ describe('Root property names', () => {
           ) {
             if (duplicatedProperties[propName]) {
               duplicatedProperties[propName].push(pageName);
-            } else {
+            } else if (!duplicatedPropertiesToIgnore.includes(propName)) {
```

Comment:
@Mottie Interested to see what your thoughts are on this. I added this line in here to skip checks on two `view` properties that were coming up as duplicates. The reason they come up as duplicates is because `evidenceTypes` and `evidenceTypesBDD` both share root property names, but these pages will never appear in the same form. This is because one shows up if `isBDD` is true and the other shows up if `isBDD` is false. Do you think it is okay to put this solution in for now? It seems like the main purpose of this test is to ensure that no fields get overwritten across pages, but in this case these will never simultaneously appear and are only being flagged as duplicates because both pages technically are in the form. I'm not sure what the other solution would be other than changing the names of the properties in one of the pages, but then I would have to adjust logic in other places to check for both the 526 properties and BDD properties.

Reply:
I think this test is fine. It's a unit test to check for problems; but I haven't seen the updated form schema.

My main concern is what happens once the `isBDD` stops reporting as `true`. Will the data still align with the `evidenceTypes`? The values are set as `enum`, right?